WalletWasabi.Fluent/Views/Wallets/Send/TransactionPreviewView.axaml

```diff
@@ -60,60 +60,59 @@
+          <MultiBinding Converter="{x:Static BoolConverters.And}">
+            <Binding Path="TransactionHasChange" />
+            <Binding Path="IsPayJoin" Converter="{x:Static BoolConverters.Not}" />
+          </MultiBinding>
+        </DockPanel.IsVisible>
+        <Button Classes="plain"
+                Command="{Binding AvoidChangeCommand}"
+                ToolTip.Tip="View suggestions"
+                DockPanel.Dock="Right"
+                Margin="10 0">
+          <Viewbox Height="20">
+            <PathIcon Data="{StaticResource edit_arrow_back_regular}" Foreground="{DynamicResource SystemAccentColor}" />
+          </Viewbox>
+        </Button>
 
-              <StackPanel Spacing="10">
-                <c:PreviewItem Icon="{StaticResource timer_regular}"
-                               Text="Bitcoin miners will work hard to confirm your transaction within"
-                               IsVisible="{Binding !IsCustomFeeUsed}">
-                  <TextBox Classes="selectableTextBlock" Text="{Binding ConfirmationTimeText, FallbackValue=~20 minutes }" />
-                </c:PreviewItem>
-                <c:PreviewItem Icon="{StaticResource paper_cash_regular}"
-                               Text="for an additional charge of">
-                  <TextBox Classes="selectableTextBlock" Text="{Binding FeeText, FallbackValue=235 sats (≈55.34 USD)}" DockPanel.Dock="Left" />
-                </c:PreviewItem>
-              </StackPanel>
-            </DockPanel>
+        <c:PreviewItem Icon="{StaticResource incognito_filled}"
+                       Text="You will receive change from this transaction"
+                       IsVisible="{Binding TransactionHasChange}"
+                       Content="View suggestions to improve privacy by avoiding change"/>
+      </DockPanel>
+      <Separator IsVisible="{Binding #avoidChangeRow.IsVisible}" />
 
       <DockPanel>
         <Button Classes="plain"
                 Command="{Binding AdjustFeeCommand}"
-                IsVisible="{Binding AdjustFeeAvailable}"
                 ToolTip.Tip="Change transaction fee or confirmation time"
                 DockPanel.Dock="Right"
                 Margin="10 0">
+          <Button.IsVisible>
+            <MultiBinding Converter="{x:Static BoolConverters.And}">
+              <Binding Path="AdjustFeeAvailable" />
+              <Binding Path="IsCustomFeeUsed" Converter="{x:Static BoolConverters.Not}" />
+            </MultiBinding>
+          </Button.IsVisible>
           <Viewbox Height="20">
             <PathIcon Data="{StaticResource adjust_fee_regular}" Foreground="{DynamicResource SystemAccentColor}" />
           </Viewbox>
         </Button>
 
         <StackPanel Spacing="10">
           <c:PreviewItem Icon="{StaticResource timer_regular}"
-                         Text="Bitcoin miners will work hard to confirm your transaction within">
+                          Text="Bitcoin miners will work hard to confirm your transaction within"
```

Comment:
This text looks weird although out of scope for this PR. Maybe we could change it to something simple or technical and remove 'work hard` part.

Reply:
Keep in mind we are aiming at the non advanced users, a "technical" wording might not be understandable for everyone. Although the current one sounds a bit technical. If I stop an average person on the street I am sure they would not know what a bitcoin miner does.

Feel free to tweak the wording in a separate PR.

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/6757